diff --git a/Examples/CondorTutorial/Applications/ImageRegTool/CMakeLists.txt b/Examples/CondorTutorial/Applications/ImageRegTool/CMakeLists.txt
index 7fd3948..b77a92d 100644
--- a/Examples/CondorTutorial/Applications/ImageRegTool/CMakeLists.txt
+++ b/Examples/CondorTutorial/Applications/ImageRegTool/CMakeLists.txt
@@ -28,10 +28,6 @@ ADD_EXECUTABLE(ImageRegTool
 INSTALL_TARGETS(/bin ImageRegTool)
 
 TARGET_LINK_LIBRARIES (ImageRegTool
-ITKIO
-ITKMetaIO
-ITKBasicFilters
-ITKStatistics
-ITKNumerics
+${ITK_LIBRARIES}
 )
 
diff --git a/Examples/CondorTutorial/Applications/ImageResample/CMakeLists.txt b/Examples/CondorTutorial/Applications/ImageResample/CMakeLists.txt
index 0303455..247e287 100644
--- a/Examples/CondorTutorial/Applications/ImageResample/CMakeLists.txt
+++ b/Examples/CondorTutorial/Applications/ImageResample/CMakeLists.txt
@@ -8,4 +8,4 @@ ELSE(ITK_FOUND)
 ENDIF(ITK_FOUND)
 
 ADD_EXECUTABLE (ImageResample main.cxx)
-TARGET_LINK_LIBRARIES(ImageResample ITKIO ITKBasicFilters)
\ No newline at end of file
+TARGET_LINK_LIBRARIES(ImageResample ${ITK_LIBRARIES})
\ No newline at end of file
diff --git a/Examples/PixelCounter/CMakeLists.txt b/Examples/PixelCounter/CMakeLists.txt
index 55e0e2c..a44396d 100644
--- a/Examples/PixelCounter/CMakeLists.txt
+++ b/Examples/PixelCounter/CMakeLists.txt
@@ -6,11 +6,11 @@ IF(ITK_FOUND)
 ENDIF(ITK_FOUND)
 
 ADD_EXECUTABLE(PixelCounter main.cxx)
-TARGET_LINK_LIBRARIES(PixelCounter BatchMakeUtilities ITKCommon ITKIO)
+TARGET_LINK_LIBRARIES(PixelCounter BatchMakeUtilities ${ITK_LIBRARIES})
 
 # Add application to generate spheres to test the Pixel Counter
 ADD_EXECUTABLE(GenerateSpheres generateSpheres.cxx)
-TARGET_LINK_LIBRARIES(GenerateSpheres ITKCommon ITKIO)
+TARGET_LINK_LIBRARIES(GenerateSpheres ${ITK_LIBRARIES})
 
 FIND_PROGRAM( PixelCounterExec PixelCounter 
   NAMES PixelCounter PixelCounter.exe
diff --git a/Testing/Code/CMakeLists.txt b/Testing/Code/CMakeLists.txt
index bbeb3ec..d763d76 100644
--- a/Testing/Code/CMakeLists.txt
+++ b/Testing/Code/CMakeLists.txt
@@ -5,7 +5,7 @@ Project(BatchMakeTest)
 SET(BATCHMAKE_TESTS ${CXX_TEST_PATH}/BatchMakeTests)
 
 ADD_EXECUTABLE(BatchMakeMemoryTest BatchMakeMemoryTest.cxx)
-TARGET_LINK_LIBRARIES(BatchMakeMemoryTest itksys ITKIO)
+TARGET_LINK_LIBRARIES(BatchMakeMemoryTest itksys ${ITK_LIBRARIES})
 
 IF(DASHBOARD_SUPPORT)
 
diff --git a/Utilities/CMakeLists.txt b/Utilities/CMakeLists.txt
index df5c626..f7b27d4 100644
--- a/Utilities/CMakeLists.txt
+++ b/Utilities/CMakeLists.txt
@@ -54,7 +54,7 @@ SET(BatchMakeUtilities_SRCS
 #  Build BatchMakeUtilities Library
 
 ADD_LIBRARY(BatchMakeUtilities ${BatchMakeUtilities_SRCS} ${BatchMakeUtilities_HDRS})
-TARGET_LINK_LIBRARIES(BatchMakeUtilities ITKCommon ITKIO itksys)
+TARGET_LINK_LIBRARIES(BatchMakeUtilities ${ITK_LIBRARIES})
 
 IF(USE_FLTK)
   TARGET_LINK_LIBRARIES(BatchMakeUtilities ${FLTK_LIBRARIES} FLU)
diff --git a/Utilities/XcedeLib/CMakeLists.txt b/Utilities/XcedeLib/CMakeLists.txt
index 3a51703..3fb4d03 100644
--- a/Utilities/XcedeLib/CMakeLists.txt
+++ b/Utilities/XcedeLib/CMakeLists.txt
@@ -21,4 +21,4 @@ SET(XCEDELIB_SRCS
    XnatCatalog.h)
 
 ADD_LIBRARY(XcedeLib ${XCEDELIB_SRCS})
-TARGET_LINK_LIBRARIES(XcedeLib itklibxml2 bmcurl ITKIO)
\ No newline at end of file
+TARGET_LINK_LIBRARIES(XcedeLib itklibxml2 bmcurl ${ITK_LIBRARIES})
\ No newline at end of file
